BasiliskII/src/Unix/ether_unix.cpp

```diff
@@ -273,8 +273,8 @@ bool ether_init(void)
 	}
 #endif
 	else {
-		net_if_type = NET_IF_SHEEPNET;
```

Comment:
Does it mean that NET_IF_SHEEPNET is not longer possible to select? Since there's no command line flag for it anymore?